sdk/eventhub/Azure.Messaging.EventHubs.CheckpointStore.Blob/src/BlobPartitionManager.cs

```diff
@@ -91,6 +93,7 @@ await foreach (var response in ContainerClient.GetBlobsAsync(options).ConfigureA
                 }
 
                 ownershipList.Add(new InnerPartitionOwnership(
+                    fullyQualifiedNamespace,
```

Comment:
I think we should add a try catch block around the foreach loop because `ContainerClient.GetBlobsAsync(options)` can throw an error while fetching the list of blobs.
```
try {
 await foreach (var response in ContainerClient.GetBlobsAsync(options).ConfigureAwait(false))
            {
                -
                -
            }
            return ownershipList;
  } catch(err) {
    Log($"Error occurred while fetching the list of blobs: '{ err }'");
    throw err;
}
```

I know this is not related to your current PR but something to keep in mind

Reply:
Just took a note. That's something I definitely should revisit when updating the error handling and the Event Processor logging.